src/main/pg/gyrodev.c

```diff
@@ -42,6 +58,23 @@
 #define GYRO_2_EXTI_PIN NONE
 #endif
 
+#ifdef MPU_ADDRESS
+#define GYRO_I2C_ADDRESS MPU_ADDRESS
+#else
+ // 0 == AUTO
+#define GYRO_I2C_ADDRESS 0
+#endif
+
+// gyro alignments
+
+#ifndef GYRO_1_ALIGN
+#define GYRO_1_ALIGN            CW0_DEG
```

Comment:
I see also DEFAULT used as value (STM32H750 uses CUSTOM which fails build)

Reply:
Will need to review CUSTOM